src/fondant/pipeline/compiler.py

```diff
@@ -137,23 +138,45 @@ def _patch_path(base_path: str) -> t.Tuple[str, t.Optional[DockerVolume]]:
         if local it patches the base_path and prepares a bind mount
         Returns a tuple containing the path and volume.
         """
+
+        def is_remote_path(path: Path) -> bool:
+            """Check if the path is remote."""
+            _path = str(path)
+            prefixes = set(known_implementations.keys()) - {"local", "file"}
+            return any(_path.startswith(prefix) for prefix in prefixes)
```

Comment:
Wouldn't it be easier to swap this check and check if it's local instead?

Reply:
Not sure. The remote prefixes are well defined by the `known_implementations`. The local paths can be more complex (start with prefix, without prefix, absolute path, relative path, ...). Thought a check for a remote path is easier.